.github/workflows/automatic-release.yml

```diff
@@ -31,6 +35,8 @@ jobs:
 
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          persist-credentials: false
```

Comment:
the checkout action also configures git by default, this is a problem because we want to set the user with our token instead, I tested removing it and it does not work. Proof:

Commit removing it: https://github.com/inpsyde/reusable-workflows/pull/87/commits/ef4a166c218b08cb7891138bd1f16e1fad04d371
Corresponding test: https://github.com/inpsyde/mercury-blocks/actions/runs/6744106204/job/18333268117

Here is [the documentation](https://github.com/actions/checkout#:~:text=persist%2Dcredentials%3A%20%27%27) regarding this config. Current default is true.

Also here is an issue / discussion around it https://github.com/actions/checkout/issues/485